fog/test-client/src/config.rs

```diff
@@ -35,6 +35,20 @@ pub struct TestClientConfig {
     #[structopt(long, env)]
```

Comment:
If this is required with the measure_after_deadline parameter, should we use structopt required_if? Also, I think the comment about failing fast should note that this is set by measure_after_deadline.

Reply:
I don't think its actually required in the code.

Previously I tried tying these fields together more tightly and that turned out not to be what we wanted, so I'd rather have a new option that's completely separate from the continuous option.

It's not clear we're going to use `fail_fast_on_deadline: false` in any tests or deployment anymore, so it might eventually go away